.travis.yml

```diff
@@ -7,4 +7,4 @@ before_install:
 script:
 - docker run  todo_test_image tests/test_view_model.py
 - docker run --env SECRET_KEY=test_secret_key --env TRELLO_API_KEY=this_is_trello_api_key_123 --env TRELLO_TOKEN=and_this_is_the_trello_token_56789 --env BOARD_ID=5fb51906cc7d395b1b145b0e todo_test_image tests/test_integration.py
-- docker run --env SECRET_KEY=$SECRET_KEY --env TRELLO_API_KEY=$TRELLO_API_KEY --env TRELLO_TOKEN=$TRELLO_TOKEN --env BOARD_ID=$BOARD_ID todo_test_image tests_e2e/test_selenium.py
```

Comment:
The selenium test shouldn't be using the BOARD_ID env var, it should be creating and using its own temporary board.

Reply:
The selenium test doesn't use BOARD_ID from `.env` file. Have removed it from the command `docker run --env` options